packages/os/whippet.service

```
@@ -0,0 +1,15 @@
+[Unit]
+Description=Bottlerocket D-Bus listener for network events
```

nit: doesn't necessarily need to be Bottlerocket-specific

```suggestion
Description=D-Bus listener for network events
```